### ISSUE TITLE
Assessment of levels of commitment per campaign

### ISSUE THREAD

**Issue by bloodmandarin** [Mindful-Meerkats/meekathon#15]
It will be interesting to see how the different numbers for "emails opened", "links clicked", "surveys completed" and "challenges fulfilled" vary over time. I have started the evaluation in the dropbox, so no reason to start this from the beginning again.

**Comment by orschiro** [comment]
@bloodmandarin 

What's the exact filename so that I can find it easily?

Thanks!

~ Robert

**Comment by bloodmandarin** [comment]
"4 Levels of Success", but that was just a working title.

**Comment by orschiro** [comment]
@bloodmandarin 

Following our taxonomy from yesterday, I would invite you to add anything related to the development of the product (i.e. meekathon) to the repository. 

Does that make sense?

**Comment by bloodmandarin** [comment]
No, I don't understand what you mean by taxonomy. Also, the file types are not all usable with Github.

**Comment by orschiro** [comment]
> No, I don't understand what you mean by taxonomy

As we said that everything related to a product is hosted on Github. Consequently also all files that are used for the evaluation. 

> Also, the file types are not all usable with Github.

Yes, they are. You can add whatever file you like.
